TELCODOCS-1092: GPU sharing methods (#67322)
openshift/openshift-docs: modules/nvidia-gpu-virtualization-with-gpu.adoc

Reviewer: Same question for this module regarding attribution and https://developer.nvidia.com/blog/improving-gpu-utilization-in-kubernetes/

Author: Rewrote: 

NVIDIA vGPU allows for direct access to a single physical GPU from virtual machines (VMs). You can create virtual GPUs that can be shared by VMs across the enterprise and accessed by other devices.

This capability combines the power of GPU performance with the management and security benefits provided by vGPU. Additional benefits provided by vGPU includes proactive management and monitoring for your VM environment, workload balancing for mixed VDI and compute workloads, and resource sharing across multiple VMs.

// Module included in the following assemblies:
//
// * architecture/nvidia-gpu-architecture-overview.adoc

:_content-type: CONCEPT
[id="nvidia-gpu-virtualization-with-gpu_{context}"]
= Virtualization with vGPU

> NVIDIA vGPU allows for direct access to a single physical GPU from virtual machines (VMs). You can create virtual GPUs that can be shared by VMs across the enterprise and accessed by other devices.

Reviewer 2: ```suggestion
Virtual machines (VMs) can directly access a single physical GPU using NVIDIA vGPU. You can create virtual GPUs that can be shared by VMs across the enterprise and accessed by other devices.
```
Reworded to avoid "allow" language:
"Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”."
